src/assets/images/index.js

```diff
@@ -0,0 +1,6 @@
+import beds from "../images/bed.jpg";
+import chair from "../images/chair.jpg";
+import coffeeTable from "../images/coffe-table.jpg";
+import sofa from "../images/sofa.jpg";
+
+export { beds, chair, coffeeTable, sofa };
```

Comment:
If possible shift the images to the `public` folder.

Reply:
will shift once P0 submission is done